core/src/test/scala/akka/persistence/cassandra/journal/CassandraIntegrationSpec.scala

```diff
@@ -104,6 +105,12 @@ import akka.persistence.cassandra.journal.CassandraIntegrationSpec._
 
 class CassandraIntegrationSpec extends CassandraSpec(config) with ImplicitSender with WordSpecLike with Matchers {
 
+  private def stopAndWaitUntilTerminated(ref: ActorRef) = {
```

Comment:
Wondering why these are needed and both you and @chbatey found it so. Something not shut down cleanly or purely related to tests and we should add it to `CassandraSpec`.

Reply:
It's the tag writer actor that is watching the PersistentActor, and it will warn if a new actor with same persistentId is started before the previous is terminated, which was what this test was doing.